### ISSUE TITLE
Docs: Loading Spinner

### ISSUE THREAD

**Issue by jeannewalters** [SAP/fundamental#396]
Page Title - **Loading Spinner**
A loading spinner informs the user of an ongoing operation. Only one busy indicator should be shown at a time.  The aria-hidden attribute is used to hide and show the element.

**Loader Element**
The loading element is used to display the loading indicator animation.

**Usage with Other Elements**
The loading indicator is most often used within another component to indicate the loading state. A container (even ui-, app-, and page-level) can be “activated” by setting aria-busy="true".

The spinner should be included inside the container. Visibility can be toggled in relation to the aria-busy attribute. They should always be opposites, i.e, if currently loading, section[aria-busy="true"], .fd-spinner[aria-hidden="false"], once the content is loaded, toggle to false and true respectively.
